src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/actions/AspectAction.java

```diff
@@ -84,15 +84,15 @@ else if (ownedElement instanceof CallBehaviorAction) {
                             }
                         }
                     }
-                    if (!aspectFound) {
-                        AspectRemedyAction ara = new AspectRemedyAction(aspected, aspect);
-                        ara.run();
-                    }
+//                    if (!aspectFound) {
```

Comment:
In the future, remove unused code instead of commenting out. Version control already maintains history of previous implementations.